### PR TITLE
[bugfix] Set number format to 4 decimal place.

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- set number format to 4 decimal place.

### CODE REVIEW

File: controllers/message.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class MessageController {
         throw new Error('insufficient balance');
       }
 
-      const newBalance = balance - price;
+      const newBalance = parseFloat((balance - price).toFixed(4));
```
Comment (platfowner) [inline controllers/message.js:47]: question) do we need parseFloat()?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: controllers/message.js (hunk 1 above)
Comment (jiyoung-an) [reply controllers/message.js:47]: .toFixed method returns a string, so to store the result as a float, we need to parse the string!

Comment (platfowner) [reply controllers/message.js:47]: ah, right, thanks!

**Comment by jiyoung-an** [comment]
Thank you for review @platfowner! now merging...

**Pull request merged**